### DISCUSSION TITLE
In RA mode and client mode, the KUR running CMP has errors

### DISCUSSION THREAD

**Discussion by XYQ202** [Keyfactor/ejbca-ce#58] [Q&A]
Referring to the openssl cmp case given by your official website, whether it is RA MODE or client mode, the registration certificate (IR) can be successful, but it will fail when updating the certificate (KUR), and it has never been successful. What's going on here, I need to configure something special on the CA UI of EJBCA, can I give some help? The configuration of CMP aliasas given by the official website is too simple, and there is always an error in KUR request, and no specific reason has been found.

The error is as follows:
![image](https://user-images.githubusercontent.com/97773232/177671047-1612c3ed-1b7a-4979-9b72-ba14156f1e5e.png)

There is no error in the command used, it should be that some special configuration is required on the CA UI

**Comment by primetomas** [comment]
Your screenshot shows IR failing not KUR as you describe. Discussion #57 have some great troubleshooting advice, usually the log shows in fairly clear text what needs to be done. KUR must be enabled in the CMP alias.
https://github.com/Keyfactor/ejbca-ce/discussions/57
